src/gui/mzroll/peakdetectiondialog.cpp

```diff
@@ -305,7 +305,11 @@ void PeakDetectionDialog::inputInitialValuesPeakDetectionDialog() {
 
         // EIC extraction windows ppm value that is set in the main
         // window is been set to the GUI
-        compoundPPMWindow->setValue(mainwindow->getUserPPM());
+        compoundPPMWindow->setValue(mainwindow->getUserMassCutoff()->getMassCutoff());
+        string userMassCutoffType=mainwindow->getUserMassCutoff()->getMassCutoffType();
+        compoundPPMWindow->setSuffix(QApplication::translate("PeakDetectionDialog", &userMassCutoffType[0], 0));
```

Comment:
There should be another box in the peak detection dialog to choose the unit. Synced with the one on mainwindow.

Reply:
This one need further discussion.